### PR TITLE
Feat/add unsubscribe all event listener

### PR DESCRIPTION
**Pull request by fajarism**
Adding a function to unsubscribe (remove) all registered event listeners called "unsubscribeAllEventListener". 
This function can be used to unsubscribe all event listeners on react cleanup effect.
This should fix multiple unintended onSuccess function calling.

## Summary by CodeRabbit

- **Refactor**
	- Renamed the function for unsubscribing from all event listeners to improve naming consistency and clarity.

### CODE REVIEW

**Review by kelvnwong — APPROVED** [review]

**Comment by bastian-purrer** [comment]
@kofimokome has this been merged?

**Comment by kofimokome** [comment]
> @kofimokome has this been merged?

No, it has not been merged.  Let me have a look

File: src/index.ts [hunk 1]
```diff
@@ -53,11 +54,16 @@ const onError: IOnError = (callback) => {
   });
 };
 
+const unsubscribeAllEventListener: IUnsubscribeAllEventListener = () => {
```
Comment (kofimokome) [inline src/index.ts]: can you make the function name plural `unsubscribeAllEventListeners`?

File: src/index.ts [hunk 2]
```diff
@@ -53,11 +54,16 @@ const onError: IOnError = (callback) => {
...
+  EventRegister.removeAllListeners();
+}
+
 export {
   configureHumanID,
   logIn,
   onError,
   onSuccess,
   onCancel,
+  unsubscribeAllEventListener,
```
Comment (kofimokome) [inline src/index.ts:67]: You have not used this function  anywhere 🤷

**Comment by kofimokome** [comment]
Hi @fajarism , Please can you fix the conflicts with your branch?

**Comment by fajarism** [comment]
@kofimokome I have fixed the conflict and change the function name according to your feedback.
Please have a look, thanks.

**Pull request merged**